helm_deploy/calculate-journey-variable-payments/templates/_envs.tpl

```diff
@@ -145,4 +145,7 @@ env:
   - name: FEATURE_FLAG_INCLUDE_RECONCILIATION_MOVES
     value: "{{ .Values.env.FEATURE_FLAG_INCLUDE_RECONCILIATION_MOVES }}"
 
+  - name: IMPORT_REPORTS_BACKDATE_ENABLED
```

Comment:
So it occurred to me we need to switch this off as there may well situations where we miss days e.g. a major service outage beyond a day whereby we resort back to a paper based system ergo potentially no electronic move data.